src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/content-item-thumbnail/content-item-thumbnail.component.ts

```diff
@@ -95,4 +114,18 @@ export class ContentItemThumbnailComponent extends BaseBrowserItemComponent<Cont
     event.path[0].src = BROKEN_IMAGE_URL;
   }
 
+  private fetchContentById(id: string): Observable<ISearchResult<Content>> {
```

Comment:
Extract this fetch method and the fetch in the basket component ts, create file in lib in angular projects (think about a name)